lib/AppInfo/Application.php

```diff
@@ -168,6 +169,24 @@ public function checkAndEnableCODEServer() {
 			$absoluteUrl = $urlGenerator->getAbsoluteURL($relativeUrl);
 			$wopi_url = $absoluteUrl . '?req=';
 
+            $CODEStatusService = $this->getContainer()->query(CODEStatusService::class);
```

Comment:
The repo uses tabs instead of spaces :wink:

Reply:
> The repo uses tabs instead of spaces wink

Got it. Corrected. IDE claims that it adheres to the current styling of the project, but seems like it failed to do that. :)